packages/zipkin/test/trace.test.js

```diff
@@ -417,11 +417,11 @@ describe('Tracer', () => {
     expect(rootTracerId.traceId.length).to.eql(32);
   });
 
-  it('should create new TraceId when joining None', () => {
+  it('should create new TraceId when joining none TraceId value', () => {
     const {recorder, ctxImpl} = setupTest();
     const tracer = new Tracer({recorder, ctxImpl});
 
-    const newTraceId = tracer.join(None).getOrElse();
+    const newTraceId = tracer.join(null);
```

Comment:
I don't think we should allow join on null should we?

Reply:
Yes, we don't.  So that is why in such case, it creates a new root id.  Should we assumes null will never pass in?  This test case just to ensure that join method works for all none TraceId value, especially null.